org.civicrm.reqangular/src/common/models/instances/contact-instance.js

```diff
@@ -1,10 +1,22 @@
 define([
-    'common/modules/models-instances',
-    'common/models/instances/instance'
+  'common/modules/models-instances',
+  'common/models/instances/instance',
+  'common/services/api/contact',
 ], function (instances) {
-    'use strict';
+  'use strict';
 
-    instances.factory('ContactInstance', ['ModelInstance', function (ModelInstance) {
-        return ModelInstance.extend({});
+  instances.factory('ContactInstance', ['ModelInstance', 'api.contact',
+    function (ModelInstance, ContactAPI) {
+      return ModelInstance.extend({
+
+        /**
+         * Finds the contacts who are managed this contact
+         *
+         * @return {Promise}
+         */
+        leaveManagees: function () {
+          return ContactAPI.leaveManagees(this.id);
```

Comment:
It could happen that this contact instance is not on the DB yet (for example if you are about to call `.create()` on it), thus it could be that it doesn't have an `id`. So please check that `id` exists before you call the api